worlds/AutoWorld.py

```diff
@@ -394,10 +400,10 @@ def pre_fill(self) -> None:
         pass
 
     def fill_hook(self,
-                  progitempool: List["Item"],
-                  usefulitempool: List["Item"],
-                  filleritempool: List["Item"],
-                  fill_locations: List["Location"]) -> None:
+                  progitempool: List[_T_Item],
+                  usefulitempool: List[_T_Item],
+                  filleritempool: List[_T_Item],
+                  fill_locations: List[_T_Loc]) -> None:
```

Comment:
Is this right? If you use the type variable `_T_Item` here, that means that only this world's items are in the list.
Don't these lists have items from all the worlds?